.eslintrc

```diff
@@ -14,7 +14,7 @@
     "sourceType": "module"
   },
   "rules": {
-    "semi": ["always"],
+    "semi": 1,
```

Comment:
Noticed this semi-colon option was incorrect ... guessed that 1 (warning) would be best.